frontend/packages/core/src/AppLayout/drawer.tsx

```diff
@@ -135,64 +98,99 @@ const Group: React.FC<GroupProps> = ({
 }) => {
   const childrenList = React.Children.toArray(children);
 
+  const [openList, setListOpen] = React.useState(false);
+
+  const anchorRef = React.useRef(null);
+
+  const handleToggle = () => {
+    setListOpen(!openList);
+  };
+
+  const handleClose = event => {
+    if (anchorRef.current && anchorRef.current.contains(event.target)) {
+      return;
+    }
+    setListOpen(false);
+  };
+
+  let headingPath = window.location.pathname.replace("/", "").split("/")[0];
```

Comment:
similar to what's happening in Link, want to change the background color of the selected heading based on the current url so that the selected state persists even when you refresh the page